src/Workspaces/CoreTest/SerializationTests.cs

```diff
@@ -39,10 +39,14 @@ public void TestNameSimplificationAnnotationSerialization()
         [Fact]
         public void VersionStamp_RoundTripText()
         {
-            using var writerStream = new MemoryStream();
-            using var writer = new ObjectWriter(writerStream);
             var versionStamp = VersionStamp.Create();
-            versionStamp.WriteTo(writer);
+
+            using var writerStream = new MemoryStream();
+
+            using (var writer = new ObjectWriter(writerStream, leaveOpen: true))
+            {
+                versionStamp.WriteTo(writer);
+            }
 
             using var readerStream = new MemoryStream(writerStream.ToArray());
             using var reader = ObjectReader.TryGetReader(readerStream);
```

Comment:
💡 This should leave open since the containing using statement will already dispose of `readerStream`